justfile

```diff
@@ -403,27 +403,33 @@ db-migrate +ARGS:
 web-ingestion-up: build-ingestion-service
     #!/usr/bin/env bash
     set -eux -o pipefail
+    compose="$(command -v podman-compose || command -v docker-compose)"
     rm -rf "$RUST_WORKSPACE/web-api/assets"
     mkdir -p "$RUST_WORKSPACE/web-api/assets"
-    ln -s "../../../$FLUTTER_WORKSPACE/example/assets/smbert_v0001/smbert.onnx" "$RUST_WORKSPACE/web-api/assets/model.onnx"
+    ln -s "../../../$FLUTTER_WORKSPACE/example/assets/smbert_v0001/smbert-quantized.onnx" "$RUST_WORKSPACE/web-api/assets/model.onnx"
```

Comment:
Not sure why these changes are here but for sure we don't want to start using the quantized model here.

Reply:
Because there is no `smbert.onnx` file there is just the mock or the quantize model.

Reply:
Ok, we did change the smbert archive without bumping the version number so the data wasn't downloaded correctly.